rust/processor/src/db/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod common;
```

Comment:
pub again, no? otherwise how will SDK use any of this

Reply:
Done